dotcom-rendering/docs/interactives/working-with-interactives.md

```diff
@@ -0,0 +1,34 @@
+# Working with Interactives
+
+Interactive is a Design type for rich content pieces - typically with custom
+Javascript for interactive effects.
+
+DCR aims to support interactive development in a variety of ways.
+
+_Note, this document is a work in progress and more will be added shortly._
+
+## Templating
+
+Interactives support placeholders for key article metadata. This is particularly
+useful for immersive interactives, which do not come with a standard top-meta
```

Comment:
are we calling immersive interactives "full page interactives" now or am I thinking of something else?

Reply:
Yeah, I thought I'd update this once the new design is fully live. But it will have to change to that (or whatever we call it).